go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v1.0.9
+	github.com/rancher/eks-operator v1.0.9-rc1
```

Comment:
Is this going backwards in versions?

Reply:
@dramich We mistakenly tagged v1.0.9 in eks-operator. I would like to delete that tag and will do so if this PR is merged.

Reply:
It's been out for 2 weeks, that's way too long to drop a tag